simulation_parameters/microbe_stage/biomes.json

```diff
@@ -1796,8 +1888,8 @@
           "Ambient": 0
         },
         "hydrogensulfide": {
-          "Amount": 0,
-          "Density": 0,
+          "Amount": 17500,
```

Comment:
And also in coastal? It is very much totally intentional game design (and maybe realism) that hydrogen sulfide is not everywhere so I highly question a PR adding sulfur chunks totally reworking the hydrogen sulfide gameplay.

Reply:
Yeah, fair enough. I'll remove this bit.